src/JWT/JWTBuilder/Builder.cs

```diff
@@ -0,0 +1,257 @@
+﻿using System;
+using JWT.JWTBuilder.Enums;
+using JWT.JWTBuilder.Helper;
+using JWT.JWTBuilder.Models;
+using JWT.Serializers;
+
+namespace JWT.JWTBuilder
+{
+    /// <summary>
+    /// Build an Decode JWT for with a Fluent-API.
+    /// </summary>
+    public class Builder
```

Comment:
I'd call it JwtBuilder withing main JWT project since it so far doesn't add new dependencies

Reply:
The reflection stuff is new. This is why I upgrade to .net standard 2.0.

Reply:
So you think we don't need the sub-namespace?

Reply:
I'd not have sub-namespaces until there are too many classes in the root or there is a strict need such as grouping/separation